src/org/zaproxy/zap/model/DefaultValueGenerator.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.model;
+
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.httpclient.URI;
+
+import net.htmlparser.jericho.FormControlType;
+
+/**
+ *   Default implementation of the DefaultValueGenerator
+ *
+ *   @since 2.6.0
+ *   @return the default String value for each control type

```

Comment:
@cdlaguna and I applied the changes for all of your latest comments. 
On this comment we just moved the return tag, not the entire javadoc. Is this what you were asking? Thanks!